portal-impl/src/com/liferay/portlet/asset/service/permission/AssetCategoryPermission.java

```diff
@@ -61,6 +62,8 @@ public static boolean contains(
 		throws PortalException {
 
 		if (actionId.equals(ActionKeys.VIEW) &&
+			(category.getVocabularyId() !=
+				AssetVocabularyConstants.INCOMPLETE_VOCABULARY_ID) &&
```

Comment:
Maybe I am missing something, but if you use a fixed ID for a missing vocabulary during import, it means that there can only be one "dummy" vocabulary. What would happen if I import categories from different vocabularies (without including the vocabularies themselves).